ui/src/nova-ui-kit/global.scss

```
@@ -89,5 +89,5 @@
 }
 
 .bubble-label {
-  @include body-number-small();
+  @include bubble-label();
```

Stylelint (17.12.0)

[error] Unexpected parentheses in argumentless mixin "bubble-label" call (scss/at-mixin-argumentless-call-parentheses)

(scss/at-mixin-argumentless-call-parentheses)